app/experimenter/targeting/constants.py

```diff
@@ -142,6 +142,36 @@ def __post_init__(self):
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+FIRST_RUN_NEW_PROFILE_HAS_PIN_NEED_DEFAULT_WINDOWS_1903 = NimbusTargetingConfig(
+    name=(
+        "First start-up users on Windows 10 1903 (build 18362) or newer, with a "
+        "new profile, needing default w/ pin"
+    ),
+    slug="first_run_new_profile_need_default_has_pin",
+    description=(
+        "First start-up users (e.g. for about:welcome) on Windows 1903+, "
+        "with a new profile, needing default w/ pin"
+    ),
+    targeting=(
+        "{first_run} && os.windowsBuildNumber >= 18362 && {has_pin} && {new_profile} && "
+        "{need_default}".format(
+            first_run=FIRST_RUN.targeting,
```

Comment:
might be worth reusing `FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903`  similar to how its used in `FIRST_RUN_NEW_PROFILE_NEED_DEFAULT_WINDOWS_1903_PREFER_MOTION` for common checks